doc/nrf/libraries/caf/buttons_pm_keep_alive.rst

```diff
@@ -0,0 +1,26 @@
+.. _caf_buttons_pm_keep_alive:
+
+CAF: Power manager keep alive module for buttons
+################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |buttons_pm_keep_alive| of the :ref:`lib_caf` (CAF) is responsible for generating :c:struct:`keep_alive_event` event on any configured button press.
+
+Configuration
+*************
+
+To use the module, you must enable the following Kconfig options:
+
+* :kconfig:`CAF_BUTTONS_PM_KEEP_ALIVE` - This option enables the |buttons_pm_keep_alive| it is enabled by default if :kconfig:`CAF_BUTTONS_PM_KEEP_ALIVE` and :kconfig:`CONFIG_CAF_BUTTONS` is enabled.
```

Comment:
Something seems wrong here. btw. maybe we could simplify:

Set the :kconfig:`CONFIG_CAF_BUTTONS_PM_KEEP_ALIVE` Kconfig option to enable the |buttons_pm_keep_alive|.
The option is enabled by default if  :kconfig:`CONFIG_CAF_POWER_MANAGER` and :kconfig:`CONFIG_CAF_BUTTONS` are enabled.`